packages/client/src/App.js

```diff
@@ -1,24 +1,19 @@
 import React from 'react';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
 import logo from './logo.svg';
 import './App.css';
+import { LandingPage } from './containers/LandingPage/LandingPage.Container';
+import { PageNotFound } from './containers/PageNotFound/PageNotFound.Container';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Router>
+        <Routes>
+          <Route path="/" element={<LandingPage />} />
+          <Route path="/*" element={<PageNotFound />} />
```

Comment:
Shouldn't this be just `*` to be a catch all?

Reply:
Yes, fixed! Thanks @miltonhultgren